datasquare.go

```diff
@@ -16,7 +16,7 @@ type dataSquare struct {
 
 func newDataSquare(data [][]byte, treeCreator TreeConstructorFn) (*dataSquare, error) {
 	width := int(math.Ceil(math.Sqrt(float64(len(data)))))
-	if int(math.Pow(float64(width), 2)) != len(data) {
+	if width*width != len(data) {
```

Comment:
I've sneaked in another unrelated change here: `int(math.Pow(float64(width), 2))` is just a complicated way of computing `width*width`.

Reply:
Saw that, and approve.